NEXT documentation (#1273)
neo4j/docs-cypher: modules/ROOT/pages/appendix/gql-conformance/supported-optional.adoc

| xref:clauses/limit.adoc[`LIMIT`], xref:clauses/order-by.adoc[`ORDER BY`]
| Cypher requires using the xref:clauses/with.adoc[`WITH`] clause, which GQL does not.

| GQ20

> | `NEXT` statement

Reviewer: ```suggestion
| Advanced linear composition with NEXT
```

| xref:queries/composed-queries/sequential-queries.adoc[]

> | The GQL standard includes a `YIELD` clause for its `NEXT` statement.
> The purpose of this `YIELD` clause is to ensure compatible behavior with SQL as it uses double quotes for delimiting identifiers.
> Cypher on the other hand uses double quotes for strings and backticks for quoting and thus having a `YIELD` clause does not provide additional use in Cypher.

Reviewer: ```suggestion
| The GQL standard includes a `YIELD` clause for its `NEXT` statement which Cypher does not implement.
```
